From: Patrick Brunschwig <patrick@enigmail.net>
Date: Wed, 27 Sep 2017 08:24:18 +0200
Subject: added missing empty line for the text/rfc822-headers section of
 Memoryhole messages

(cherry picked from commit ae9cc596ddd3d615a2ad447556c952aab47745bb)
---
 package/mimeEncrypt.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/mimeEncrypt.js b/package/mimeEncrypt.js
index 876399a..5b7fc43 100644
--- a/package/mimeEncrypt.js
+++ b/package/mimeEncrypt.js
@@ -296,7 +296,7 @@ PgpMimeEncrypt.prototype = {
 
     if (this.cryptoMode == MIME_ENCRYPTED && this.enigSecurityInfo.sendFlags & Ci.nsIEnigmail.ENCRYPT_HEADERS) {
       w += 'Content-Type: text/rfc822-headers; protected-headers="v1"\r\n' +
-        'Content-Disposition: inline\r\n' +
+        'Content-Disposition: inline\r\n\r\n' +
         visibleHdr +
         "\r\n--" + this.encHeader + "\r\n";
     }
